test/metabase/api/card_test.clj

```diff
@@ -1035,6 +1035,14 @@
                :type    "question"}
               (mt/user-http-request :crowberto :post 200 "card" (card-with-name-and-query (mt/random-name))))))))
 
+(deftest create-card-with-metric-type
+  (mt/with-model-cleanup [:model/Card]
+    (testing "can create a metric card"
+      (is (=? {:dataset false
+               :type    "metric"}
+              (mt/user-http-request :crowberto :post 200 "card" (assoc (card-with-name-and-query (mt/random-name))
```

Comment:
should we be checking that a metric card's query conforms to the rules around metrics (must have 1 aggregation)?